src/octotree.js

```diff
@@ -167,28 +182,36 @@ $(document).ready(() => {
       errorView.show(err);
     }
 
-    function toggleSidebarAndSave() {
-      store.set(STORE.SHOWN, !isSidebarVisible(), () => {
-        toggleSidebar();
-        if (isSidebarVisible()) {
-          tryLoadRepo();
-        }
-      });
-    }
-
     function toggleSidebar(visibility) {
       if (visibility !== undefined) {
         if (isSidebarVisible() === visibility) return;
         toggleSidebar();
       } else {
         $html.toggleClass(SHOW_CLASS);
         $document.trigger(EVENT.TOGGLE, isSidebarVisible());
+        if (isSidebarVisible()) {
+          tryLoadRepo();
```

Comment:
Why reload repo here?

Reply:
The first load is necessary when opening the sidebar in the layout mode.